src/table/column-filter.scss

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-column-filter {
+  @include iui-dropdown;
+  padding: $iui-baseline $iui-sm;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+
+  > .iui-toggle-switch {
+    align-self: flex-start;
+  }
+
+  .iui-icon-overlay {
+    position: relative;
+    width: 100%;
+
+    > .iui-input {
+      padding-right: $iui-l * 2;
+    }
+
+    > .iui-borderless {
+      position: absolute;
+      right: 0;
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+  }
+
+  > .iui-input-container {
+    align-self: flex-end;
```

Comment:
Is this needed? Because toggling it seems to do nothing.

Reply:
`iui-input-container` is no more 🪦

Reply:
You are correct, it seems like it is not needed.  Removed.